_component_design/buttons.md

```diff
@@ -295,23 +242,25 @@ A drop-down button is preferred when multiple actions are available, with no des
 
 <section class="static-section" markdown="1">
 
-<div class="hxRow" markdown="1">
+<div class="hxRow"  markdown="1">
```

Comment:
Why are spaces being added here? No need to have two spaces (though the code doesn't really care).